<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="vensakp51" name="jdbcConnection" tests="1" failures="1" timestamp="29 Jul 2016 06:02:45 GMT" time="0.274" errors="0">
  <testcase name="setJDBCConnection" time="0.254" classname="jdbcPrograms.ConnectJDBC">
    <failure type="java.sql.SQLSyntaxErrorException" message="Table &amp;apos;employeemgmt.persons&amp;apos; doesn&amp;apos;t exist">
      <![CDATA[java.sql.SQLSyntaxErrorException: Table 'employeemgmt.persons' doesn't exist
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:686)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:663)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:653)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:108)
	at com.mysql.cj.jdbc.ConnectionImpl.execSQL(ConnectionImpl.java:2061)
	at com.mysql.cj.jdbc.ConnectionImpl.execSQL(ConnectionImpl.java:2014)
	at com.mysql.cj.jdbc.StatementImpl.executeQuery(StatementImpl.java:1424)
	at jdbcPrograms.ConnectJDBC.setJDBCConnection(ConnectJDBC.java:52)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- setJDBCConnection -->
</testsuite> <!-- jdbcConnection -->
